Bottom: 3b409221551a281a042ecfa60a6de3b76b1b69ec
Top:    16955da58186ad2a47b744c887a3e51c014ce063
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 17:51:46 +0100

Refresh of hid-gtk-replace-deprecated-fun

---

diff --git a/src/hid/gtk/gui-dialog.c b/src/hid/gtk/gui-dialog.c
index 93cea47..1770bba 100644
--- a/src/hid/gtk/gui-dialog.c
+++ b/src/hid/gtk/gui-dialog.c
@@ -76,7 +76,7 @@ ghid_dialog_input (const char * prompt, const char * initial)
     gtk_entry_set_text (GTK_ENTRY (entry), initial);
 
   gtk_entry_set_activates_default (GTK_ENTRY (entry), TRUE);
-  gtk_box_pack_start_defaults (GTK_BOX (vbox), entry);
+  gtk_box_pack_start (GTK_BOX (vbox), entry, TRUE, TRUE, 0);
 
   content_area = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
   gtk_container_add (GTK_CONTAINER (content_area), vbox);
